### ISSUE TITLE
Accessibility: When zooming in to 175% the contents panel receives the style display="none"

### ISSUE THREAD

**Issue by nhsbsa-wicoo** [nhsbsa/nhsbsa-digital-playbook#496]
Accessibility: Scrollable content areas cannot be accessed using a keyboard

## Summary

When pages are zoomed into 175%, the 'Content' disappears from view.

## Expected behaviour

The 'Content' panel should reflow into a stacked or collapsable list.

## User impact

This issue primarily affects:

- Users with low vision

## WCAG 2.2 Success Criteria
1.4.10 Reflow
